dev/customer_testing/pubspec.yaml

```diff
@@ -19,4 +19,43 @@ dependencies:
   string_scanner: 1.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   term_glyph: 1.2.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
-# PUBSPEC CHECKSUM: 5fe8
+dev_dependencies:
+  # See packages/flutter_test/pubspec.yaml for why we're pinning this version.
```

Comment:
i think we should remove this comment (both from here and from devicelabb/pubspec.yaml where I suspect this was copied from), since really update_packages.dart is the only place that really documents why we pin anything these days.